base_ical/__manifest__.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2023 Hunki Enterprises BV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl-3.0)
+
+{
+    "name": "Readonly publishing of calendars",
+    "summary": "Provide (readonly) .ics URLs to calendar-like models",
+    "version": "15.0.1.0.0",
+    "development_status": "Alpha",
+    "category": "Tools",
+    "website": "https://github.com/OCA/server-backend",
```

Comment:
Should this land in https://github.com/OCA/calendar instead?

Reply:
Not sure how we want to sort it. You can publish leaves => holidays. You can publish activities as TODO => social(?). You can publish a calendar => calendar. You can also publish attendance => attendance. server-backend might fit as general place.

You will probably use it in your calendar application but I'm not sure if this is the correct criteria.